apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V30__Create_preadjudicated_claims_tables.sql

```diff
@@ -0,0 +1,73 @@
+/*
+ * Creates the RDA FISS pre-adjudicated claims tables required by the application.
+ * https://confluenceent.cms.gov/pages/viewpage.action?spaceKey=MPSM&title=MVP+0.1+Data+Dictionary+for+Replicated+Data+Access+API
+ */
+
+/*
+ TODO - Determine whether we want to store tables in public schema or create a new one.
+ For now the table names are prefixed with PreAdj (pre-adjudicated).
+ */
+
+/*
+TODO - what tablespace should we use as a default?
+*/
+${logic.tablespaces-escape} SET default_tablespace = fhirdb_ts2;
+
+create table "PreAdjFissClaims"
+(
+    "dcn"               varchar(23) not null,
+    "hicNo"             varchar(12) not null,
+    "currStatus"        char(1)     not null,
+    "currLoc1"          char(1)     not null,
+    "currLoc2"          varchar(5)  not null,
+    "medaProvId"        varchar(13),
+    "totalChargeAmount" decimal(11, 2),
+    "receivedDate"      date,
+    "currTranDate"      date,
+    "admitDiagCode"     varchar(7),
+    "principleDiag"     varchar(7),
+    "npiNumber"         varchar(10),
+    "mbi"               varchar(13),
+    "mbiHash"           varchar(64),
+    "fedTaxNumber"      varchar(10),
+    "lastUpdated"       timestamp with time zone,
+    constraint "PreAdjFissClaims_pkey" primary key ("dcn")
+)
+/*
+TODO - should we have a tablespace for this table or just use the default?
+${logic.tablespaces-escape} tablespace "preadjfissclaims_ts"
+*/
+;
+
+/*
+ * Index the MedicareBeneficiaryIdentifier (MBI) hash column for mbi searches.
+ * The search is used by partners to find claims for specific beneficiaries.
+ * Since this is a frequent operation, the hash is indexed.
+ */
+
+create index "PreAdjFissClaims_mbi_hash_idx"
+    on "PreAdjFissClaims" ("mbiHash");
+
+/*
+ * Columns with leading underscore in the name are unique to our database.
+ * Other columns map directly to fields in the RDA API result message.
+ *
+ * The _priority column is a 0 based index indicating the order in which the codes
+ * appear in the RDA API result message for the claim.
```

Comment:
That's an interesting convention. I'm curious what other folks think of it?

Personally, aside from my deep inner desire to categorize **ALL THE THINGS**, I can't come up with a reason that it'd actually be helpful. Given that, I'd probably leave it out.

Reply:
When we return the procedure codes in the FHIR resource, I'm guessing the order in which we return them doesn't matter, so in that case it's not needed. FISS does make a distinction between the first procedure code (the primary) and all the others (2-25), so if for any reason we want to map back to the primary procedure code we'll need the `_priority` value.

Reply:
Oh darn it, looks like I highlighted the wrong rows.

Can we pretend I attached this note to the comment about the underscore prefix, instead?

(FWIW, I would preserve order as much as possible.)

Reply:
Oh I missed that comment.  @brianburton I thought the RDA API team was defining the `_priority` field?  I thought the only field we're generating are `mbiHash` and `lastUpdatedBy`.

Reply:
The RDA schema has an internal field `_priority` but it is not directly passed through the API.  I am basically reconstructing it on our end based on the order of the procedure codes in the API response data.  So it is a derived field on our side.

Reply:
@karlmdavis there is some benefit to distinguishing our internal column names from those in the RDA API in case we run into a conflict with new fields later.  Especially with a young, rapidly evolving API like the RDA.

That said I did not use the underscore for lastUpdated or mbiHash because I wanted to follow conventions used in BFD so as conventions go this PR already violates it.

Reply:
> That said I did not use the underscore for lastUpdated or mbiHash because I wanted to follow conventions used in BFD so as conventions go this PR already violates it.

The underscore convention gets more confusing to me because of this. Would make sense to me to remove it and be diligent about documenting which columns come directly from the RDA API.

Reply:
Agreed.  I'll drop it.

Reply:
Latest commit removes the underscore.